site/src/system.md

```diff
@@ -1,5 +1,5 @@
 
-# `system`
+# Cross-platform: `system`
```

Comment:
I'd like to reserve "cross" for cross-compilation if possible. What do you think of

```suggestion
# Multi-platform: `system`
```